app/Http/Controllers/PublicProfileController.php

```diff
@@ -29,6 +29,23 @@ public function index()
             ->with('speakers', $users);
     }
 
+    public function search(Request $request)
+    {
+        $users = User::search($request->search_query)
+            ->orderBy('name', 'asc')->get();
+
+        // Since Scout searches can only perform rudimentary where clauses,
+        // we must filter search results to only validly public profiles.
+        $filteredUsers = $users->filter(function ($user) {
```

Comment:
Got it. 👍🏻 

And yah, what I meant by indexing profiles, not users, is just a mental model sort of thing; that is, indexing *for the sake of showing the profile*, which means we could totally exclude indexing non-public-profile people. But again, I can't even remember if Scout can do that. So I'm sure this is fine for now.